blocks/extras/extensions/plot.py

```diff
@@ -6,6 +6,7 @@
 try:
     from bokeh.plotting import (curdoc, cursession, figure, output_server,
                                 push, show)
+    from bokeh.util.platform import is_notebook
```

Comment:
I've got 0.9.0 on my machine - and it looks like the function was added recently to `` bokeh/bokeh/util/platform.py
`` : 

```
def is_notebook():
    """ Test whether we are inside an IPython notebook.
    Returns
        True if we are inside a notebook, otherwise False
    """
   try:
       get_ipython()
       return True
   except NameError:
       return False
```
I guess it depends on your view of updating dependencies...